multinode-demo/bootstrap-leader.sh

```diff
@@ -47,6 +47,16 @@ $program \
 pid=$!
 oom_score_adj "$pid" 1000
 
-setup_fullnode_staking 127.0.0.1 "$bootstrap_leader_id_path" "$bootstrap_leader_staker_id_path"
+leader_rotation=true
+for i in "$@"; do
+if [[ ${i} == --no-leader-rotation ]] ; then
```

Comment:
nits: 
* [ ] indent `if` block
* [ ] Use `=` instead of `==` (consistency with the `=` below)
* [ ] `$i` instead of `${i}` (less syntax cruft)